apps/settings/src/components/AppStoreSidebar/AppDeployOptionsModal.vue

```diff
@@ -0,0 +1,322 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcModal v-if="show"
+		label-id="form-name"
+		@close="() => $emit('update:show', false)">
+		<div class="modal__content">
+			<h2 id="form-name">
+				{{ t('settings', 'Advanced deploy options') }}
+			</h2>
+			<p class="description" style="text-align: center;">
+				{{ configuredDeployOptions === null ? t('settings', 'Edit ExApp deploy options before installation') : t('settings', 'Configured ExApp deploy options. Can be set only during installation') }}.
+				<a href="https://docs.nextcloud.com/server/latest/admin_manual/exapps_management/AdvancedDeployOptions.html">Learn more</a>
```

Comment:
Also translate the "Learn more".

```suggestion
				{{ configuredDeployOptions === null
					? t('settings', 'Edit ExApp deploy options before installation.')
					: t('settings', 'Configured ExApp deploy options. Can be set only during installation.')
				}}.
				<a href="https://docs.nextcloud.com/server/latest/admin_manual/exapps_management/AdvancedDeployOptions.html">
					{{ t('settings', 'Learn more') }}
				</a>
```